layouts/feed.njk

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Looks correct, but have we tested this works upstream in for posts on the X-GOVUK website?

Reply:
I’ve tested this by running `npm pack` and then copying the zip file to the x-govuk.github.io repo, and referencing it there from the `package.json`. Seems to work!